week-3/reading-list/script.js

```diff
@@ -21,3 +21,23 @@ const books = [
   },
 ];
 
+const ul = document.querySelector('#reading-list');
+for (const book of books) {
+  const li = document.createElement("li");
+  ul.appendChild(li);
+  //Display title of the book
+  const bookName = document.createElement("p");
+  bookName.innerText = book.title + "-------"+book.author;
+  li.appendChild(bookName);
+ //Display author of the book
+ //Display title of the book
+ const img = document.createElement("img");
+img.src = book.bookCoverImage;
+ li.appendChild(img);
+
+ if(book.alreadyRead===false){
```

Comment:
In general we don't tend to _compare_ values with `true` or `false`.

An `if` statement takes an expression which evaluates to a truthy value, so another way of writing `if (book.alreadyRead === false)` is `if (!book.alreadyRead)` - we tend to favour the `!` approach over the `=== false` approach because it's more concise and we know `alreadyRead` is already a boolean.